bitbucket/resource_branching_model.go

```diff
@@ -311,6 +311,7 @@ func flattenBranchModel(rp *BranchModel, typ string) []interface{} {
 		"is_valid":              rp.IsValid,
 		"use_mainbranch":        rp.UseMainbranch,
 		"name":                  rp.Name,
+		"enabled":               rp.Enabled,
```

Comment:
For `production`, the BitBucket API returns an `enabled` field. We simply need to copy it from the `BranchModel` to the output. In the `develop` case, the field is not present and defaults to `false`.